source/datainjection/index.rst

```diff
@@ -1,158 +1,200 @@
 
-Fist step you need te create model, for this example we import Computer
+-  Use an existing template in the drop-down menu
 
-Clic here to create new model
+-  Manage templates to :
 
-.. image:: images/create_model.png
+   -  Create a new template
+   -  Modify an existing template
+   -  Delete an existing template
+   -  Use an existing template
 
-And fill form
+Creating a template
+~~~~~~~~~~~~~~~~~~~
 
-.. image:: images/add_model.png
+The 1st step is to create a template if none exists.
 
-* Name : define a model Name
-* Visibility : is private or not for other user
-* Entity / sub entity : model visibility for entity
-* Comments : model comment
-* Type of data : type of data to import
-* Allow lines creation : yes or not
-* Allow lines update : yes or not
-* Allow creation of dropdowns : if dropdown value not exist, let's create It
-* Dates format : date format in CSV file
-* Allow update of existing fields : yes or not
-* Float format : float format in CSV file
-* Try to establish network connection is possible : yes or not
-* Port unicity criteria : define unicity field fir port
+-  From **Tools >Data injection**
+-  Click on **Template**
+-  Click on **Add**
 
-After model creation it's possible to :
-* Define if header is present
-* Change file delimitor : default ->  ";"
-
-Inject your CSV file
---------------------
+.. figure:: images/Datainjection-2.png
+   :alt:
 
-Send to GLPI your CSV file with computer data
+Defining the model
+^^^^^^^^^^^^^^^^^^
 
-.. image:: images/send-file.png
+This step allows you to define the main options for the extension:
 
+-  **Name**: specify the name that will be visible from the drop-down list
+-  **Visibility**: define whether it will be visible to other users or not
+-  **Entity** and **sub-entity**: the template can be visible in the creation entity only (sub-entity set to no) or in the entity and all sub-entity (sub-entity set to yes).
+-  **Comment**: Enter a comment for the template.
+-  **Type of data to import**: This is the type of data present in the file to be imported. A file only consists of data of the same type.
+-  **Allow lines creation**: tells the extension to create objects if the line does not correspond to an existing element in the database.
+-  **Allow update rows**: tells the extension to modify existing objects using the data in the file.
+-  **Header’s presence**: tells the extension whether the headings present in the file (location, type of equipment, models, etc.) should be created. Creation will also be limited by the rights of the template user.
+-  **Update existing fields**: tells the extension whether the data in the file can replace fields already filled in.
+-  **Date format**: indicates which date format is contained in the CSV file. If the format is incorrect, the date cannot be imported.
+-  **Decimal number format**: indicates the format used for the numbers in the CSV file. If the format is incorrect, the numbers cannot be imported.
```

Comment:
```suggestion
-  **Name**: the name that will be visible from the drop-down list
-  **Visibility**: defines whether it will be visible to other users or not
-  **Entity** and **sub-entity**: the template can be visible in the creation entity only (sub-entity set to no) or in the entity and all sub-entity (sub-entity set to yes).
-  **Comment**: a comment for the template.
-  **Type of data to import**: the type of data present in the file to be imported. A file only consists of data of the same type.
-  **Allow lines creation**: tells the plugin to create objects if the line does not correspond to an existing element in the database.
-  **Allow update rows**: tells the plugin to modify existing objects using the data in the file.
-  **Header’s presence**: tells the plugin whether the headings present in the file (location, type of equipment, models, etc.) should be created. Creation will also be limited by the rights of the template user.
-  **Update existing fields**: the extension whether the data in the file can replace fields already filled in.
-  **Date format**: the date format is contained in the CSV file. If the format is incorrect, the date cannot be imported.
-  **Decimal number format**: the format used for the numbers in the CSV file. If the format is incorrect, the numbers cannot be imported.
```